examples/todolist/tests/bindings/test_todolist.swift

```diff
@@ -42,3 +42,8 @@ assert(try! todo.getLast() == "bar")
 todo.addItems(items: ["bobo", "fofo"])
 assert(todo.getItems().count == 9)
 assert(todo.getItems()[7] == "bobo")
+
```

Comment:
nano nit: Can we add a comment here that this loop tests destructors?

Reply:
:+1: